BT.json

```diff
@@ -557,7 +557,7 @@
 			"wowID":"31098"
 		},
 		"Leggings of the Forgotten Protector":{
-			"prio":["Tatertotem(1)","Hunter/DPS War > Ele/Resto sham","EP/GP"],
+			"prio":["Tatertotem(1)","Chiron (1)","Doochie? (2)","Oglol (2)","Putitah (2)","Hunter/DPS War > Ele/Resto sham","EP/GP"],
```

Comment:
Prot warrior first, most lists have t6 tank legs as bis.

Reply:
im fine with prot warrs getting the legs first but that means tater wont be getting the first 2 piece which is insane for ele.

Reply:
Yeah let's do that, tater will understand gearing a tank for mit is more important overall.